18_regexp_lab/02.py

```diff
@@ -1,6 +1,17 @@
-"""
-Write a function toCamelCase(text) that returns the text camel cased
-Write a function to_underscore(text) that returns the text with 
-underscores between words
-"""
-
+import re
+
+#To Camel case
+def toCamelCase(text):
+    allWords = re.split(r'_', text)
+    first = allWords[0]
+    return first + ''.join([word.title() for word in allWords[1:]])
+
+#From camel case
+def fromCamelCase(text):
+    return re.sub(r'[A-Z]',
+        lambda m: '_' + m.group(0).lower()
+        , text)
+
+
+print toCamelCase('no_more_shall_we_part')
+print  fromCamelCase('noMoreShallWePart')

```

Comment:
Nice, though your roCamelCase has not really used the power of regexes. You could have used regular split as well.